Makefile

```diff
@@ -20,25 +20,31 @@ all: clean bundle ## will build a binary for all platforms
 	@export GOOS=freebsd GOARCH=386; $(MAKE) build;
 	@export GOOS=freebsd GOARCH=amd64; $(MAKE) build;
 build:
+	@mkdir -p build/release
 	@mkdir -p build/dist/${GOOS}-${GOARCH} && \
     echo "[${GOOS}-${GOARCH}] build started" && \
 		CGO_ENABLED=0 go build \
 			-ldflags="-s -w" \
 			-o build/dist/${GOOS}-${GOARCH}/theme${EXT} \
 			github.com/Shopify/themekit/cmd/theme && \
+		zip -qj build/release/${GOOS}-${GOARCH}.zip \
```

Comment:
Minor, but I think it's useful to use the long-form flags in scripts, i.e. `zip --quiet --junk-paths`, so it's self-documenting.